lib/post_creator.rb

```diff
@@ -397,6 +399,27 @@ def unarchive_message
     end
   end
 
+  def remove_useless_quotes
```

Comment:
I think this should be done in a post processor since we don't want more overhead when creating a post.

Reply:
Also do we need this to be a public method?